code/development-env/docker/README.md

```diff
@@ -1,11 +1,24 @@
 # Setup local development environment using docker-compose
 
+## Pre-requisites
+
+* [Install Homebrew](https://brew.sh/)
+* [Install Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac).
+* [Install kubectl with Homebrew](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos)
+* Use official kubectl, not Docker's: `brew link --overwrite kubernetes-cli`
```

Comment:
Weird - I'm pretty sure I just did a brew install kubernetes-cli and got the right thing! What version of Docker for Mac are you using? Presumably latest? - https://github.com/docker/for-mac/issues/2890
Do you have Kubernetes support turned on in Docker?

This doesn't feel like a standard problem so I'd prefer this line wasn't in the standard dependencies. Could you refine it to explain what the problem is that it solves and then you can say "Check if you have this problem and fix it by running..."

Reply:
Hi Gary, I believe it's order dependent.  I installed Docker for Mac first, and so there wouldn't have been anything in /usr/local/bin/kubectl, and so the Docker for Mac installer linked it to /Applications/Docker.app/Contents/Resources/bin/kubectl (without overwriting anything, and so different to the situation in https://github.com/docker/for-mac/issues/2890).  Then when installing kubectl with Homebrew, afterwards, Homebrew detects the link at /usr/local/bin/kubectl and so leaves it alone.  The `brew link --overwrite kubernetes-cli` persuades Homebrew to overwrite the link to the Docker for Mac version.  I'll update the README to make it clearer.

Reply:
Ahhh, makes sense, thanks for clarifying. I guess when I first installed Docker Desktop it probably didn't distribute kubectl